modules/@ergonode/authentication/src/components/Layout/Login.vue

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE for license details.
+ */
+<template>
+    <main class="login">
+        <section class="login__body">
+            <span
+                v-if="showReleaseInfo"
+                class="release-info"
+                v-text="showReleaseInfo" />
+            <FluidBlob />
+            <slot />
+        </section>
+        <Footer space-between>
+            <div class="login__footer-info">
+                <Select
+                    class="footer-info__select"
+                    :value="value"
+                    :options="languageOptions"
+                    :size="smallSize"
+                    @input="onValueChange" />
+                <LinkButton
+                    :title="$t('authentication.layout.help')"
+                    @click.native="onHelpClicked" />
+            </div>
+            <IconLogoName class="login__footer-logo" />
```

Comment:
It would be better to add possibility of modifying width / height by props - it's SVG and styles are going to overwrite initial sizes:
```

        width="171"
        height="48"
        viewBox="0 0 171 48"

```

Reply:
There's no need to do that. You're forcing me to refactor places unrelated to the task

Reply:
But it's overwrites original behaviour, isn't that better to add better support to this icon?